src/local.c

```diff
@@ -56,7 +56,10 @@ check_localdate(void)
         /* *INDENT-OFF* */
         fprintf(stdout, "%s", ANSI_BOLD_ON);
         fprintf(stdout, "%s", ANSI_COLOR_CODE_FG);
-        fprintf(stdout, "Local data is older than two weeks, use --update to update it.\n\n");
+        fprintf(stdout, "Local data is older than two weeks, attempting to automatically update it...\n");
+        if (update_localdb(0))
```

Comment:
This has output which ends up with the bold red, even though it probably shouldn't be.
Could we reset the foreground color and disable bold before this `if`?

Alternatively, we may want to consider switching from red (looks like an error) to a yellow color to more visually it's a warning/heads-up, and nothing actually went wrong.

Without introducing any additional colors, I think this would be better:

```c
if (difftime > (60 * 60 * 24 * 7 * 2)) {
    /* *INDENT-OFF* */
    fprintf(stdout, "Local data is older than two weeks, attempting to automatically update it...\n");
    if (update_localdb(0)) {
        fprintf(stdout, "%s%s", ANSI_BOLD_ON, ANSI_COLOR_CODE_FG);
        fprintf(stdout, "Failed to automatically update local data.\n");
        fprintf(stdout, "%s%s", ANSI_COLOR_RESET_FG, ANSI_BOLD_OFF);
    }
    fprintf(stdout, "\n");
    /* *INDENT-ON* */
}
```
> PS: I'm not much of a C person, is there any reason why multiple ANSI codes weren't done like that before? :thinking: 

### Initial UX (unchanged)

![image](https://user-images.githubusercontent.com/22801583/229616202-7b055e4e-12df-4a4f-8cc9-2cb96dfe85c9.png)
> When a first does their first command and no local DB exists.

### Current behavior for outdated DB

![image](https://user-images.githubusercontent.com/22801583/229616456-0e504d8f-8c90-4aed-93b9-6f39102eeb17.png)

### PR behavior for outdated DB

![image](https://user-images.githubusercontent.com/22801583/229616729-d3a1000d-a482-4573-a545-b2a391b035e0.png)
> Red tends to indicate an error or in the case before, that action was required. But if this is no longer the case, this should change.

### Proposed behavior for outdated DB

![image](https://user-images.githubusercontent.com/22801583/229616597-ae62bf2b-8eea-4ea4-b875-d97b7fd5dfb9.png)
> Colors/width now aligns with the initial user experience.